notebooks/Evaluation - noise.ipynb

```diff
@@ -0,0 +1,2261 @@
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "results = []\n",
+    "for checkpoint, args in zip(checkpoints,run_args):\n",
+    "    print(\"------------ RUNNING \", checkpoint, \" -----------------\")\n",
+    "    print(args)\n",
+    "    args[\"batch_size\"] = 256\n",
+    "    args[\"data_dir\"] = \"./data/\"\n",
+    "    args[\"use_train_model\"] = False\n",
+    "    args[\"load_path\"] = checkpoint\n",
+    "    \n",
+    "    if args[\"seed\"] == seed_wanted:\n",
+    "        checkpoint_path = os.path.join(args[\"load_path\"])\n",
+    "        checkpoint = torch.load(checkpoint_path,map_location='cuda:0')\n",
+    "        load_model = (checkpoint[\"train_model\"] if args[\"use_train_model\"] else checkpoint[\"eval_model\"])\n",
+    "        _net_builder = net_builder(args[\"net\"],False,{})\n",
+    "        _eval_dset = SSL_Dataset(name=args[\"dataset\"], train=False, data_dir=args[\"data_dir\"], seed=args[\"seed\"])\n",
+    "        eval_dset = _eval_dset.get_dset()\n",
+    "        inv_transf = _eval_dset.inv_transform\n",
+    "        transf = _eval_dset.transform\n",
+    "        net = _net_builder(num_classes=_eval_dset.num_classes, in_channels=_eval_dset.num_channels)\n",
+    "        net.load_state_dict(load_model)\n",
+    "        if torch.cuda.is_available():\n",
+    "            net.cuda()\n",
+    "        net.eval()\n",
+    "                        \n",
+    "        eval_loader = get_data_loader(eval_dset, args[\"batch\"], num_workers=1)\n",
+    "        label_encoding = _eval_dset.label_encoding\n",
+    "        dB_value_list = []\n",
+    "        resuts_run = []\n",
+    "        \n",
+    "        for P_noise_dB in range(P_noise_dB_min, P_noise_dB_max, dP_noise_dB):\n",
+    "            print(\"------------ PREDICTING TESTSET ----------------- - Noise value [dB]\", P_noise_dB)\n",
+    "            \n",
+    "            \n",
+    "            images, labels, preds = [],[],[]\n",
+    "            with torch.no_grad():\n",
+    "                for image, target in tqdm(eval_loader):\n",
+    "                    image_preprocessed=[]\n",
+    "                    for idx,img in enumerate(image):\n",
+    "                        img=255 * inv_transf(img.transpose(0,2).cpu().numpy())\n",
+    "                        img_orig=img\n",
+    "                        for n in range(img.shape[0]):\n",
+    "                            img[n]=(0.5 + 0.75 * torch.rand(size=(1,))) * img[n]\n",
```

Comment:
In the paper it said 0.75 to 1.25 , but you do 0.5 to 1.25

Reply:
Corrected. Thanks.

Reply:
I don't think you pushed your changes. At least didn't change here